services/ext/api.go

```diff
@@ -394,12 +394,6 @@ func (api *PublicAPI) SetInstallationName(installationID string, name string) er
 	return api.service.messenger.SetInstallationName(installationID, name)
 }
 
-// Communities returns a list of communities that are stored
```

Comment:
I wonder if it makes sense to return "seralized" communities from other endpoints, such as `JoinedCommunities`